pkg/cli/cli.go

```diff
@@ -144,8 +144,14 @@ func newCLI(options ...Option) (*CLI, error) {
 func (c *CLI) buildCmd() error {
 	c.cmd = c.newRootCmd()
 
+	var uve config.UnsupportedVersionError
+
 	// Get project version and plugin keys.
-	if err := c.getInfo(); err != nil {
+	switch err := c.getInfo(); {
+	case err == nil:
+		// do nothing if it's nil
+	case !errors.As(err, &uve):
+		// Ignore 3-alpha error
 		return err
```

Comment:
@Adirio I think 3-alpha is a special case; in general we would want to treat non-existent project versions as actual errors right?
```suggestion
	case !errors.As(err, &uve) || uve.Version.String() != "3-alpha":
		// Ignore 3-alpha version error only.
		return err
```

Reply:
The condition would be like this right?
```suggestion
	case errors.As(err, &uve) && uve.Version.Compare(config.Version{Number: 3, Stage: stage.Alpha}) == 0:
		// Ignore 3-alpha error
	default:
		return err
```